test-openjdk/pom.xml

```diff
@@ -231,12 +237,7 @@
             </activation>
             <dependencies>
                 <dependency>
-                    <!--Need tools.jar in compilation classpath. JDK 1.7 no longer has it there by default: -->
-                    <groupId>com.sun</groupId>
-                    <artifactId>tools</artifactId>
-                    <version>${maven.compiler.target}</version>
-                    <scope>system</scope>
-                    <systemPath>${java.home}/../lib/tools.jar</systemPath>
+
```

Comment:
I htink this remans valid for jdk8

Reply:
Hmm, I did not notice any issues with ubi 8 jdk8. I will try rhel 7 jdk8 to make sure that does not break too.

Reply:
fair enogh then. If you tried, its ok. I hav enoticed several issues with tools.jar over years, so was doubel checking it. No need to rerun in el7, the packaging is same